unique_toolkit/unique_toolkit/content/service.py

```diff
@@ -350,6 +351,76 @@ def _trigger_upload_content(
 
         return Content(**created_content)
 
+    def send_download_content_from_chat_request(
+        self, content_id: str, chat_id: str
+    ) -> requests.Response:
+        url = f"{unique_sdk.api_base}/content/{content_id}/file"
+        if chat_id:
+            url = f"{url}?chatId={chat_id}"
+
+            # Download the file and save it to the random directory
+        headers = {
+            "x-api-version": unique_sdk.api_version,
+            "x-app-id": unique_sdk.app_id,
+            "x-user-id": self.event.user_id,
+            "x-company-id": self.event.company_id,
+            "Authorization": "Bearer %s" % (unique_sdk.api_key,),
+        }
+
+        return requests.get(url, headers=headers)
+
+    def download_content_from_chat_to_file_by_id(
```

Comment:
I would say download_content_from_chat is descriptive enough.

Reply:
changed to `download_content_to_file_by_id` as when `chat_id` is not given the content is downloaded from the knowledge base